docs/reference/constrained.rst

```diff
@@ -53,6 +53,7 @@ Methods
    ConstrainedQuadraticModel.iter_violations
```

Comment:
I think these should all be `~ConstrainedQuadraticModel.` to be consistent with `QuadraticModel` and `BinaryQuadraticModel`.